automl/google/cloud/automl_v1beta1/tables/tables_client.py

```diff
@@ -2642,6 +2643,14 @@ def predict(
                 The `model` instance you want to predict with . This must be
                 supplied if `model_display_name` or `model_name` are not
                 supplied.
+            params (Dict[str, str]): 
+                Additional domain-specific parameters, any string must be up to 
+                25000 characters long.
+                ``feature_importance`` - (boolean) Whether
+                [feature\_importance][[google.cloud.automl.v1beta1.TablesModelColumnInfo.feature\_importance]
```

Comment:
Curious if there is documentation / spec about how to how to write docstring with links.
E.g.,
- why "\" in "feature\_importance" is needed,
- what does "(-s)" mean, and
- why different from markdown, [text][[link] instead of [text][link] is used.

Reply:
This formatting is from the proto comments. I believe it's some flavor of markdown that makes the comments render nicely on cloud.google.com. See https://github.com/googleapis/googleapis/tree/master/google/cloud/automl/v1 and https://cloud.google.com/automl/docs/reference/rpc/google.cloud.automl.v1.

When you need to add docstrings, please follow the [Google Python Style Guide](http://google.github.io/styleguide/pyguide)